cmds/bmp/bms_command_test.go

```diff
@@ -70,13 +75,69 @@ var _ = Describe("bms command", func() {
 			Expect(validate).To(BeTrue())
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		Context("bad BmsCommand", func() {
+			Context("no deployment file", func() {
+				BeforeEach(func() {
+					options = cmds.Options{
+						Verbose: false,
+						Deployment: "",
+					}
```

Comment:
My intention is to test empty string, but after a second thought, I think it is not so meaningful here, so removed it. (The next case will verify fake file name)